src/main/java/net/javadiscord/javabot/listener/SuggestionListener.java

```diff
@@ -51,7 +51,7 @@ private boolean canCreateSuggestion(MessageReceivedEvent event) {
 		if (event.getChannelType() == ChannelType.PRIVATE) return false;
 		return !event.getAuthor().isBot() && !event.getAuthor().isSystem() && !event.getMember().isTimedOut() &&
 				event.getMessage().getType() != MessageType.THREAD_CREATED &&
-				event.getChannel().equals(Bot.config.get(event.getGuild()).getModeration().getSuggestionChannel());
+				event.getChannel().equals(Bot.config.get(event.getGuild()).getModerationConfig().getSuggestionChannel());
```

Comment:
`event.getChannel()` returns a `MessageChannelUnion` while `config.getSuggestionChannel()` returns a `TextChannel`. These are not the same. Better compare the channel ids.